install.sh

```diff
@@ -805,6 +815,14 @@ set_user_subuids_subgids() {
   fi
 }
 
```

Comment:
Add function `run_as_user` which will run a command with arguments as `sudo -u $OPT_USER command [args]`. Some commands such as `trdl add werf ...` must run via this function.